charts/svn-to-git/templates/configmap-secret.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: svn-config
```

Comment:
If multiple instances of this chart will be installed, shouldn't we append a unique identifier or release name in the name of cm and secret?